src/Illuminate/Database/Eloquent/Factories/Factory.php

```diff
@@ -469,7 +469,7 @@ protected function expandAttributes(array $definition)
     /**
      * Add a new state transformation to the model definition.
      *
-     * @param  (callable(array<string, mixed>): array<string, mixed>)|array<string, mixed>  $state
+     * @param  (callable(array<string, mixed>, \Illuminate\Database\Eloquent\Model|null=): array<string, mixed>)|array<string, mixed>  $state
```

Comment:
This is invalid syntax

Reply:
According to PHPStan (https://phpstan.org/writing-php-code/phpdoc-types#callables), the equals represents an optional parameter, but I'm not sure if that's what you're referring to.

Reply:
Ah, nice. Thanks. :)